docs/start/quickstart_group.mdx

```diff
@@ -192,11 +189,30 @@ For Step 2, select the "Creator" tab if you are coordinating the creation of the
                 be functioning for the validator(s) to stay active.
               </li>
             </ul>
+             :::info
+              If you are configuring a cluster for use within Lido's Community Stake Module (CSM),
+              in the following steps you need to:
+              <ul>
+                <li>
+                  Set the <code>validators</code> field to 1. (only 1 CSM Validator can be deployed per cluster via the launchpad). 
+                </li>
+                <li>
+                  Set the <code>withdrawal configuration</code> to <code>custom</code>.  
+                </li>
+                <li>
+                  Set <code>withdrawal address</code> to Lido’s Withdrawal Vault Address: 0xF0179dEC45a37423EAD4FaD5fCb136197872EAd9 per [Lido’s documentation](https://docs.lido.fi/deployed-contracts/holesky/).  
+                </li>
+                <li>
+                  Set the <code>fee recipient</code> to Lido’s Execution Layer Rewards Vault Address: 0xE73a3602b99f1f913e72F8bdcBC235e206794Ac8 per Lido’s [documentation](https://docs.lido.fi/deployed-contracts/holesky/).   
```

Comment:
I think showing holesky addresses here is a bad idea, I don't believe this guide is network specific anywhere other than this.